examples/statistics/plot_hist.py

```diff
@@ -0,0 +1,101 @@
+"""
+==========
+Histograms
+==========
+
+Demonstrates how to plot histograms with matplotlib.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import colors
+from matplotlib.ticker import PercentFormatter
+
+# Fixing random state for reproducibility
+np.random.seed(19680801)
+
+
+###############################################################################
```

Comment:
For reviews, this syntax in sphinx gallery is supported to convert this into text in both the html and to a text block in the jupyter notebook version of the plot.